go/exorelay/features/retry.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Retrying
+
+  As a develper building Exosphere applications
+  I want want ExoRelay to retry establishing a connection
+  So that I have to think less about the network level
+
+  Rules
+  - ExoRelay will retry connecting to Exocom with exponential decay
+  - Randomness is added to the interval to prevent multiple services from reconnecting at once 
+
+  Background:
+    Given an ExoRelay with the role "foo"
+
+  Scenario: ExoRelay attempts to establish a connection to ExoCom
+    Given Exocom is not online yet
+    When ExoRelay connects to Exocom while Exocom is booting up
+    Then it registers by sending the message "exocom.register-service" with payload:
+      """
+      { "clientName": "foo"} 
+      """
```

Comment:
Please also add a test where the connection is established, then disconnected, and Exorelay re-connects.